ci/checks/changelog.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+# Copyright (c) 2018, NVIDIA CORPORATION.
+#########################
+# cuDF CHANGELOG Tester #
+#########################
+
+# Checkout master for comparison
+git checkout --quiet master
+
+# Switch back to tip of PR branch
+git checkout --quiet current-pr-branch
+
+# Ignore errors during searching
+set +e
+
+# Get list of modified files between matster and PR branch
+CHANGELOG=`git diff --name-only master...current-pr-branch | grep CHANGELOG.md`
+# Check if CHANGELOG has PR ID
+PRNUM=`cat CHANGELOG.md | grep "$PR_ID"`
```

Comment:
Definitely a nitpick, but I've seen people combining multiple PRs into a single line item on the changelog which I'd personally like to avoid them doing. Doesn't need to block this PR being merged, but I'd like to enforce that it's a PR_ID per line.

Reply:
I’ll add this in the morning as it’s a easy change

Reply:
@kkraus14 only thing here is will this rule block people from referencing other PRs in their line, for example:

```
- PR #491 Adds script as part of a larger effort to onboard CI scripts into repo, see also #550
```

I was about to write a test that would detect that the current PR ID is only present once in the CHANGELOG, but that could cause an issue here.

Reply:
@kkraus14 did say it's a nitpick. I don't think it's crucial to enforce.

Reply:
Will leave as-is then